.github/workflows/benchmarks.yml

```diff
@@ -101,9 +110,23 @@ jobs:
           terraform_version: 1.3.7
           terraform_wrapper: false
 
+      - name: Init terraform module
+        run: |
+          AWS_REGION=${{ env.AWS_REGION }}
+          echo "TF_VAR_worker_region=$AWS_REGION" >> "$GITHUB_ENV"
+          echo "TF_VAR_run_standalone=$RUN_STANDALONE" >> "$GITHUB_ENV"
```

Comment:
If these env variable are only used within the context of this step, you can use the `env`


```yaml
      - name: Init terraform module
        run: make init
        env:
          TF_VAR_worker_region: ${{ env.AWS_REGION }}
          TF_VAR_run_standalone: ${{ env.RUN_STANDALONE }}
```

Otherwise, do you think you can set them in the top level env? `L54`

Reply:
This step sets variables for terraform, I'm not 100% sure if `terraform init` is the step than binds the ENV to variables or it happens during `terraform apply` let me double check this and update it if it works in `terraform init`.

Reply:
Updated via steps output now. Also apparently TF binds ENVs to variables at `terraform apply`.